xmodule/course_block.py

```diff
@@ -57,6 +58,17 @@
 COURSE_VIDEO_SHARING_PER_VIDEO = 'per-video'
 COURSE_VIDEO_SHARING_ALL_VIDEOS = 'all-on'
 COURSE_VIDEO_SHARING_NONE = 'all-off'
+# .. toggle_name: FEATURES['CREATE_COURSE_WITH_DEFAULT_ENROLLMENT_START_DATE']
```

Comment:
No change is being requested in this comment.

[inform] There has been an unofficial move away from using the `FEATURES` dict. We should probably have a DEPR to at least discuss it.
FYI: @feanil

Reply:
[inform] I created https://github.com/openedx/edx-platform/issues/33026 to discuss this topic. Again, no request for change.